### PR TITLE
updated hoek as per #19 issue.

### PR DESCRIPTION
**Pull request by tlivings**
required bumping to support LTS releases only

### CODE REVIEW

**Comment by sebastianvirlan** [comment]
Will this be merged?

**Comment by computnik** [comment]
+1
When is this planned to be merged and published?

File: .gitignore [hunk 1]
```diff
@@ -33,4 +33,5 @@ Thumbs.db
 .AppleDouble
 .LSOverride
 .Spotlight-V100
-.Trashes
\ No newline at end of file
+.Trashes
+package-lock.json
```
Comment (guilhermehn) [inline .gitignore]: I think it would be better to set `package-lock=false` on .npmrc so the lock file would not be generated at all

**Comment by UsainBloot** [comment]
Is there a reason why this hasn't been merged and released?

**Pull request merged**